src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,10 +30,16 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        String[] addressSections = address.split(", ");
+        this.block = new Block(addressSections[0],isPrivate);
```

Comment:
think why it makes little sense to have individual components marked as private or not (at least in the context of this application)?